### PR TITLE
[doxygen] grammar and tag fixes

### PR DESCRIPTION
**Pull request by Vyrastas**
- Replaced all instances of "the the" in comments with "the"
- Couple of Doxygen tag fixes (Border Router brief, misspelling)

### CODE REVIEW

**Comment by Vyrastas** [comment]
I'm assuming the addition of new lines at the end of some files was forced, I didn't add those manually.

File: third_party/mbedtls/repo.patched/include/mbedtls/ssl_ticket.h [hunk 1]
```diff
@@ -92,7 +92,7 @@ void mbedtls_ssl_ticket_init( mbedtls_ssl_ticket_context *ctx );
  *                  Recommended value: 86400 (one day).
  *
  * \note            It is highly recommended to select a cipher that is at
- *                  least as strong as the the strongest ciphersuite
+ *                  least as strong as the strongest ciphersuite
```
Comment (jwhui) [inline third_party/mbedtls/repo.patched/include/mbedtls/ssl_ticket.h]: We should probably leave this typo as-is, since we pull this from mbedtls directly.

**Comment by Vyrastas** [comment]
@qiangg I did a sed command to make the mass changes and then pushed it to my fork.  So somewhere in there it added the newline I guess.

**Comment by srickardti** [comment]
Hey @Vyrastas,

Thanks for pointing these errors out in driverlib. I've opened a ticket against the maintainers of driverlib so that it will be fixed in a future release.

The files you have changed do carry the Texas Instruments BSD-3-Clause header. We would prefer to keep these files as close to what we release in our SDK as possible, typos and all :). However, the BSD license allows modifications, and Google is re-licensing them here; so, I will deffer to their judgment on accepting changes. I'm not sure what the implications of changing `third_party` content is.

The changes only affect documentation, and look fine from a technical point of view.

**Comment by jwhui** [comment]
@Vyrastas, given that the `third_party` code carries a different copyright, should we limit our docs fixes to code outside `third_party`?

**Comment by Vyrastas** [comment]
@jwhui Yeah that might be best.  I wasn't aware that there was a different license involved.  I can go ahead and revert the `third_party` changes if there are no objections.

**Comment by Vyrastas** [comment]
Since there are more `third_party` updates than not, it'll be easier to just close this and create a new pull request.  Vendors can use this as a guide to correct instances of "the the" in their code.

**Pull request closed without merging**